### PR TITLE
[ENH] Updated `QueryForm` layout to clarify query parameters

### LINKED ISSUE #285: Clarify query parameter names
While we are still only handling non-session-level phenotypic information, we should update the names of the query parameters in the UI accordingly so there is no confusion about which timepoint users are querying for.

These include:
"Min/max age" -> "Min/max age at baseline"
"Diagnosis" -> "Diagnosis at baseline"
"Minimum number of sessions" -> "Minimum number of imaging sessions"

- See also https://github.com/neurobagel/bagel-cli/issues/225

TODO
- [x] have on-hover tooltip for each query parameter

### PR DESCRIPTION
**Pull request by rmanaem**
- Closes #285 

## Checklist

- [x] PR has an interpretable title with a prefix (`[ENH]`, `[FIX]`, `[REF]`, `[TST]`, `[CI]`, `[MNT]`, `[INF]`, `[MODEL]`, `[DOC]`) _(see https://neurobagel.org/contributing/pull_requests for more info)_
- [x] PR links to GitHub issue with mention `Closes #XXXX`
- [x] Tests pass
- [x] Checks pass

For new features:
- [ ] Tests have been added

For bug fixes:
- [ ] There is at least one test that would fail under the original bug conditions.

### CODE REVIEW

File: components/QueryForm.vue [hunk 1]
```diff
@@ -25,6 +25,12 @@
               @input="$emit('selectNodes', $event)"
             />
           </div>
+          <div>
+            <hr>
+          </div>
+          <b-row>
+            <h6>Phenotypic fields (at baseline)</h6>
```
Comment (surchs) [inline components/QueryForm.vue]: could you try a higher order heading? It's hard to notice that these are subsection headings - they seem to have the same font weight as the rest of the query fields in the preview: 
image

**Review by surchs — APPROVED** [review]
One :cherries: comment @rmanaem: could we make the section headings more visible,

then :cook:

File: components/QueryForm.vue (hunk 1 above)
Comment (rmanaem) [reply components/QueryForm.vue]: Yea I used `h5` first but the Phenotypic one will overflow to the next line

image

Comment (rmanaem) [reply components/QueryForm.vue]: @surchs What do you think?

Comment (surchs) [reply components/QueryForm.vue]: linebreak is fine, we can always make it prettier!

**Pull request merged**